manywheel/build_common.sh

```diff
@@ -298,6 +299,7 @@ for pkg in /$WHEELHOUSE_DIR/torch*linux*.whl /$LIBTORCH_HOUSE_DIR/libtorch*.zip;
     fi
 
     # zip up the wheel back
+    echo "$(pushd $pytorch_rootdir && git rev-parse HEAD)" > $PREFIX/build-hash
```

Comment:
I don't think this is needed. Line 154 of this file should cover this.

Reply:
Thx. Remove it.